src/nebula/NebulaAction.h

```diff
@@ -457,12 +456,14 @@ class CleanDataAction : public core::Action {
  * */
 class RandomPartitionAction : public core::DisturbAction {
 public:
-    RandomPartitionAction(const std::vector<NebulaInstance*>& metas,
+    RandomPartitionAction(const std::vector<NebulaInstance*>& graphs,
```

Comment:
Em..  If we support multi graphds,  which one the graphClient will connect?

Reply:
Ah, yeah, fixed.